src/theme/type.tsx

```diff
@@ -40,6 +40,12 @@ export function H3(props: TextProps) {
   )
 }
 
+export function H4(props: TextProps) {
+  return (
+    <TextWrapper className="headline headline-4" fontSize={20} fontWeight={500} lineHeight="28px" noWrap {...props} />
```

Comment:
is there a mismatch between figma and our design tokens? i see a `headlineSmall` in the design spec but these numbers do indeed match up 👀

Reply:
yes, it seems like our ThemedText needs an audit. [i have a task for it here](https://uniswaplabs.atlassian.net/browse/WEB-2782?atlOrigin=eyJpIjoiMTQ0MzBlNjY4N2JhNGNjYzgzMjMyNWM2NjUyODBhOGMiLCJwIjoiaiJ9)